sdk/core/Azure.Core/src/Response.cs

```diff
@@ -9,33 +9,86 @@
 
 namespace Azure
 {
+    /// <summary>
+    /// Represents the HTTP response from the service.
+    /// </summary>
     public abstract class Response : IDisposable
     {
+        /// <summary>
+        /// Gets the HTTP status code.
+        /// </summary>
         public abstract int Status { get; }
 
+        /// <summary>
+        /// Gets the HTTP reason phrase.
+        /// </summary>
         public abstract string ReasonPhrase { get; }
 
+        /// <summary>
+        /// Gets the contents of HTTP response. Returns <c>null</c> for responses without content.
+        /// </summary>
         public abstract Stream? ContentStream { get; set; }
 
+        /// <summary>
+        /// Gets the client request id that was sent to the server as <c>x-ms-client-request-id</c> headers.
+        /// </summary>
         public abstract string ClientRequestId { get; set; }
 
+        /// <summary>
+        /// Get the HTTP response headers.
+        /// </summary>
         public virtual ResponseHeaders Headers => new ResponseHeaders(this);
 
+        /// <summary>
+        /// Frees resources held by this <see cref="Response"/> instance.
+        /// </summary>
         public abstract void Dispose();
 
+        /// <summary>
+        /// Returns header value if headers is stored in the collection. If header has multiple values they are going to be joined with a comma.
+        /// </summary>
+        /// <param name="name">The header name.</param>
+        /// <param name="value">The reference to populate with value.</param>
+        /// <returns><code>true</code> if the specified header is stored in the collection, otherwise. <code>false</code></returns>
         protected internal abstract bool TryGetHeader(string name, [NotNullWhen(true)] out string? value);
 
+        /// <summary>
+        /// Returns header values if headers is stored in the collection.
+        /// </summary>
+        /// <param name="name">The header name.</param>
+        /// <param name="values">The reference to populate with values.</param>
+        /// <returns><code>true</code> if the specified header is stored in the collection, otherwise. <code>false</code></returns>
         protected internal abstract bool TryGetHeaderValues(string name, [NotNullWhen(true)] out IEnumerable<string>? values);
 
+        /// <summary>
+        /// Returns if headers is stored in the collection.
+        /// </summary>
+        /// <param name="name">The header name.</param>
+        /// <returns><code>true</code> if the specified header is stored in the collection, otherwise. <code>false</code></returns>
```

Comment:
No period after "otherwise". Should be comma.

Reply:
... and often the "otherwise" is preceded by a semicolon instead of comma, but I *think* either way is correct.